server/routes/referrals/draftReferralsListPresenter.ts

```diff
@@ -0,0 +1,23 @@
+import { DraftReferral } from '../../services/interventionsService'
+
+export default class DraftReferralsListPresenter {
+  constructor(private readonly draftReferrals: DraftReferral[]) {}
+
+  get orderedReferrals(): DraftReferralSummaryPresenter[] {
+    return this.draftReferrals
+      .sort((a, b) => (new Date(a.createdAt) < new Date(b.createdAt) ? -1 : 1))
+      .map(referral => ({
+        id: referral.id.slice(0, 8), // this is totally arbitrary, and probably meaningless to the user
```

Comment:
Is this just to make it shorter in the display?

Reply:
yeh - the table looked silly with the full id. i guess this is kind of the skeleton idea. it works but it's very rough.